### ISSUE TITLE
KeyError in metadata.py extract_catstags function

### ISSUE THREAD

**Issue by rgeronimi** [adbar/trafilatura#47]
I caught the following exception:
```
...
  File "/usr/local/Caskroom/miniconda/base/envs/myenv/lib/python3.7/site-packages/trafilatura/core.py", line 586, in bare_extraction
    docmeta = extract_metadata(tree, url, date_extraction_params)
  File "/usr/local/Caskroom/miniconda/base/envs/myenv/lib/python3.7/site-packages/trafilatura/metadata.py", line 393, in extract_metadata
    metadata['categories'] = extract_catstags('category', tree)
  File "/usr/local/Caskroom/miniconda/base/envs/myenv/lib/python3.7/site-packages/trafilatura/metadata.py", line 331, in extract_catstags
    results.append(element.attrib['content'])
  File "src/lxml/etree.pyx", line 2479, in lxml.etree._Attrib.__getitem__
KeyError: 'content'
```

I use Python 3.7.9 and Trafilatura 0.6.1 (from Pypi)

**Comment by adbar** [comment]
Hi, thanks, I just fixed it in 912b02b3cab08c232e565d007faebf3dc0808ca4.

**Comment by rgeronimi** [comment]
Great thanks!
